spec/helpers/fill_in_fields.rb

```diff
@@ -0,0 +1,245 @@
+# frozen_string_literal: true
+
+# Fillin the form files
```

Comment:
`spec/helpers` should only contain tests for helper classes.  Helpers for the test suite should be in `spec/support`.